test/core/test_capture.js

```diff
@@ -1,27 +1,71 @@
 'use strict';
 
-const test = require('tape');
+const { test, beforeEach } = require('tap');
 const runner = require('../../core').runner;
-const L = require('lodash');
+const nock = require('nock');
+const uuid = require('uuid');
 const csv = require('csv-parse');
 const fs = require('fs');
 const path = require('path');
 const { SSMS } = require('../../core/lib/ssms');
+const http = require('http');
+const WebSocket = require('ws');
 
 let xmlCapture = null;
 try {
   xmlCapture = require('artillery-xml-capture');
 } catch (e) {}
 
+beforeEach(() => nock.cleanAll());
+
 test('Capture - headers', (t) => {
-  const fn = path.resolve(__dirname, './scripts/captures-header.json');
```

Comment:
The file does not seem to be used anywhere else? should probably delete it altogether if so